assets/js/modules/analytics/components/dashboard/DashboardGoalsWidget.js

```diff
@@ -148,5 +149,6 @@ function DashboardGoalsWidget() {
 
 export default whenActive( {
 	moduleName: 'analytics',
-	fallbackComponent: AnalyticsInactiveCTA,
+	FallbackComponent: <AnalyticsInactiveCTA />,
+	IncompleteComponent: <CompleteModuleActivationCTA slug="analytics" />,
```

Comment:
See above, also applies to other similar usages.

Reply:
As above.